### PR TITLE
476 updates

### PR DESCRIPTION
**Pull request by mitogi**
## Changes proposed in this pull request:

1. Added sticky leftmost column and a max-width condition
2. Cleaned up punctuation
3. Moved table description above
4. Added back to blueprint button

!. changed use case from "burger" to "getting fast food"
2. edited table content for clarity

##considerations

had to change the dark green border on first table cell to same color as rest of table. Otherwise the dark green get overlapped with light green border when sticky column feature is scrolled.

### CODE REVIEW

File: content/methods/service-blueprint-example.md [hunk 1]
```diff
@@ -16,25 +16,28 @@ eleventyExcludeFromCollections: true
 <div class="layout--service-blueprint-example">
   <section class="category category--decide usa-section usa-prose">
     <section class="example-card grid-container">
+      <a class="usa-button usa-button--base" href="/methods/decide/service-blueprint/">Back to Service Blueprint Methods</a>
```
[the remark this answers is not in the document]
Comment (mitogi) [reply content/methods/service-blueprint-example.md:19]: changed

File: content/methods/service-blueprint-example.md [hunk 2]
```diff
@@ -16,25 +16,28 @@ eleventyExcludeFromCollections: true
...
+
       <h1>Service blueprint example</h1>
       <div class="grid-col-10">
 
         This simple [service blueprint]({{ "/methods/decide/service-blueprint/" | url }}) is an example of how you might lay out the layers of your service. There are many ways to create a service blueprint and adapt the structure to best represent and capture your context.
-      </div>
+      
+        <p class="caption">This is a table showing an example service blueprint. The top row is the journey of a customer ordering and eating at a fast food restaurant. Each column is a different step in the customer's journey. The remaining rows describe different parts of the service and the support provided at each step in the customer’s journey.</p>
```
[the remark this answers is not in the document]
Comment (mitogi) [reply content/methods/service-blueprint-example.md:26]: done

**Comment by cantsin** [comment]
Checking on status here; do we need more review @sam-couch-18f @mitogi ?

**Comment by MelissaBraxton** [comment]
Hi @mitogi  - I get a 404 when I try and use the "back to blueprint" button in the preview. Once that's fixed, it looks good to me!

**Review by MelissaBraxton — CHANGES_REQUESTED** [review]
I get a 404 when trying to use the "return to blueprint" button when I checked this out on th Pages preview. Once that's fixed, I think this'll be good to go!

**Comment by mitogi** [comment]
Updated to link for Back to service blueprint button. Should work now. Thanks for catching that, Melissa!

**Review by MelissaBraxton — APPROVED** [review]
Looks good to me!

**Pull request merged**